### PR TITLE
Update recent transactions component

### PR DESCRIPTION
**Pull request by CedrikNikita**

### CODE REVIEW

**Comment by imollov** [comment]
web version:
Screenshot 2021-06-30 at 10 58 20

**Comment by CedrikNikita** [comment]
that is why it is draft

**Comment by imollov** [comment]
> that is why it is draft

Sorry, thought it might be helpful.

File: src/popup/router/pages/Account.vue [hunk 1]
```diff
@@ -47,7 +47,17 @@ export default {
 
   .recent-transactions {
     margin-top: -10px;
-    flex-grow: 1;
```
Comment (imollov) [inline src/popup/router/pages/Account.vue:50]: This is needed so the loader and no-items message are centered vertically.

**Review by imollov — COMMENTED** [review]
View all button is broken on iOS:

Simulator Screen Shot - iPhone 12 mini - 2021-08-17 at 15 52 56

File: src/popup/router/pages/Account.vue [hunk 2]
```diff
@@ -15,38 +15,102 @@
...
+      background: variables.$color-bg-1;
+      text-decoration: none;
+      color: variables.$color-dark-grey;
+      transition: all 0.12s ease-out;
+
+      @extend %face-sans-15-medium;
+
+      .icon {
+        margin-left: 4px;
+        width: 24px;
```
Comment (imollov) [inline src/popup/router/pages/Account.vue:96]: Fix for iOS issue

```suggestion
        width: 24px;
        height: 24px;
```

**Review by imollov — APPROVED** [review]

**Pull request merged**